modules/javafx.graphics/src/test/java/test/javafx/scene/layout/RegionTest.java

```diff
@@ -1009,7 +1009,631 @@ public void testComputeChildPrefAreaHeightHonorsMinWidthOverMax() {
     }
 
     @Test
-    public void testChildMinAreaWidth() {
+    public void testChildMinAreaWidthExtensively() {
+        Pane pane = new Pane();
+
+        Region c1 = new MockBiased(Orientation.VERTICAL, 100, 200);
+        Region c2 = new MockBiased(Orientation.HORIZONTAL, 100, 200);
+        Region c3 = new MockRegion(10, 10, 100, 100, 1000, 1000);
+
+        pane.getChildren().addAll(c1, c2, c3);
+
+        /*
+         * Ensure that no fillHeight/height combinations have effect on controls that are not vertically biased:
+         */
+
+        assertEquals(12, RegionShim.computeChildMinAreaWidth(pane, c2, -1, new Insets(1), -1, false), 1e-100);
```

Comment:
might as well declare a static `computeChildMinAreaWidth()` which delegates to  `RegionShim.computeChildMinAreaWidth`...

Reply:
Yeah, I can streamline this a bit more.